### PR TITLE
Fix 2 bugs in vf package tests

### PR DESCRIPTION
**Pull request by hlhoots**
#build

### CODE REVIEW

**Comment by hlhoots** [comment]
#libby

**Review by jimblye — DISMISSED** [review]

File: dev/com.ibm.ws.kernel.boot_fat/publish/files/versionlessServer.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     
     <featureManager>
         <platform>jakartaee-10.0</platform>
-        <platform>microProfile-4.0</platform>
+        <platform>microProfile-5.0</platform>
```
Comment (jhanders34) [inline dev/com.ibm.ws.kernel.boot_fat/publish/files/versionlessServer.xml]: These still conflict.  JEE 10 goes with MP 6 and 6.1.  JEE 9 goes with MP 5.

Comment (hlhoots) [reply dev/com.ibm.ws.kernel.boot_fat/publish/files/versionlessServer.xml]: fixed

**Review by jimblye — APPROVED** [review]

**Review by jhanders34 — APPROVED** [review]

**Comment by hlhoots** [comment]
RTC rebuild due to infra issue : https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_nVDg0EQ6Ee-F88sHKWQ5jw

backup build due to power outage: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_FwTMkESOEe-ngPvsOTExmw

**Comment by hlhoots** [comment]
Good build here: https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_FwTMkESOEe-ngPvsOTExmw

Merging.

**Pull request merged**